temporian/implementation/numpy/operators/propagate.py

```diff
@@ -0,0 +1,72 @@
+"""Implementation for the Propagate operator."""
+
+
+from typing import Dict
+
+import numpy as np
+import itertools
+
+from temporian.implementation.numpy.data.event import NumpyEvent, NumpyFeature
+from temporian.implementation.numpy.data.sampling import NumpySampling
+from temporian.core.operators.propagate import Propagate
+
+
+class PropagateNumpyImplementation:
+    """Numpy implementation for the propagat operator."""
+
+    def __init__(self, op: Propagate) -> None:
+        assert isinstance(op, Propagate)
+        self._op = op

```

Comment:
`self.operator` to keep it consistent with existing ones? else if `_op` is better (which I agree on) I would change all of the existing ones to it

Reply:
I don't have preference between _op or _operator. It seems the "operator" version is currently more common in the code, so I'll update to that. However, I see value in the "_": This is a private attribute.

*Change to _operator